pkg/moq/testpackages/dotimport/service.go

```diff
@@ -1,7 +1,7 @@
 // Package dotimport addresses issue 21.
 package dotimport
 
-//go:generate moq -out service_moq_test.go -pkg dotimport_test . Service
+//go:generate moq -out service_moq_test.go -with-resets -pkg dotimport_test . Service
```

Comment:
This test was written for a specific purpose and it would be better to keep the failure signal unambiguous. Can we add a separate test specifically for testing the reset flag?